source/configure/plugins-configuration-settings.rst

```diff
@@ -505,10 +505,34 @@ Enable plugin
 | - **false**: Disables the Calls plugin on your Mattermost workspace. | - ``config.json`` setting:                |
 +----------------------------------------------------------------------+-------------------------------------------+
 
+.. config:setting:: plugins-callsrtcserveraddress
+  :displayname: RTC server port (Plugins - Calls)
+  :systemconsole: Plugins > Calls
+  :configjson:
+  :environment:
+  :description: The IP address used by the RTC server to listen on. By default the service listens on all the available interfaces.
+
+RTC server address
+~~~~~~~~~~~~~~~~~~
+
+.. include:: ../_static/badges/selfhosted-only.rst
+  :start-after: :nosearch:
+
++-----------------------------------------------------------------------------------------------------------------------+-------------------------------------------+
+| This setting controls the IP address listened on by the RTC server. All Calls traffic will be served through this IP. | - System Config path: **Plugins > Calls** |
+|                                                                                                                       | - ``config.json`` setting:                |
```

Comment:
If this config setting doesn't have an equivalent ``config.json`` setting, please populate the ``config.json`` field in this table as ``N/A``, and include the ``Environment variable`` line to the table with a value of ``N/A``.